System/google_chrome_use_cpu.1m.sh

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/env bash
+#  <xbar.title>CPU Google Chrome</xbar.title>
+#  <xbar.version>v1.0</xbar.version>
+#  <xbar.author>Alexandre Espinosa Menor</xbar.author>
+#  <xbar.author.github>alexandregz</xbar.author.github>
+#  <xbar.image>https://imgur.com/G7D2XS9</xbar.image>
```

Comment:
```suggestion
#  <xbar.image>https://imgur.com/G7D2XS9.png</xbar.image>
```

Reply:
I changed screenshot to see Google Chrome cpu usage and other app, for example QEMU, and I renamed plugin.

Also I added CPU icon (generic, for other apps not Chrome) and I used xbar vars.

Thx for your feedback!